tools/dev-release.sh

```diff
@@ -1,13 +1,43 @@
-#!/bin/sh -xe
+#!/bin/bash -xe
 # Release dev packages to PyPI
 
+Usage() {
+    echo Usage:
+    echo "$0 [ --production ]"
```

Comment:
nit: If I'm reading the code correctly, if you use `--production` you also need to include the current release version and the next version? This should be reflected here as I wouldn't have known that unless I read the code.